.github/workflows/pipeline.yml

```diff
@@ -1,55 +1,49 @@
 name: CI-Checks
-on: 
-  push:
-    branches-ignore:
-      -master
-      -release-*
-jobs:
-  check-newsfile:
+on: [push]
+jobs: 
+  check-code-style:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
           python-version: '3.7'
-      - run: python -m pip install towncrier
-      - run: python -m towncrier.check
-    # plugins:
-    #   - docker#v3.0.1:
-    #       image: "python:3.7"
-    #       mount-buildkite-agent: false
-    # branches: "!master !release-*"
-
-  # - command:
-  #     - "python -m pip install tox"
-  #     - "tox -e check_codestyle"
-  #   label: "Check Code Style"
-  #   plugins:
-  #     - docker#v3.0.1:
-  #         image: "python:3.7"
-  #         mount-buildkite-agent: false
-
-  # - command:
-  #     - "python -m pip install tox"
-  #     - "tox -e check_types"
-  #   label: "Check Types (:mypy:)"
-  #   plugins:
-  #     - docker#v3.0.1:
-  #         image: "python:3.7"
-  #         mount-buildkite-agent: false
-
-  # - wait
-
-  # - command:
-  #     - "python -m pip install -e ."
-  #     - "trial tests"
-  #   label: "Run unit tests (on SQLite)"
-  #   plugins:
-  #     - docker#v3.0.1:
-  #         image: "python:3.7"
-  #         mount-buildkite-agent: false
+      - run: python -m pip install tox
+      - run: tox -e check_codestyle
+  
+  check-types-mypy:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - run: python -m pip install tox  
+      - run: tox -e check_types
 
-  # - command:
+  run-unit-tests-sqlite:
+    needs: [check-code-style, check-types-mypy]
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - run: python -m pip install -e .
+      - run: trial tests
+  
+  # run-unit-tests-(on :postgres:):
```

Comment:
I'm not sure if Github Actions supports custom emoji, but `:postgres:` on buildkite would translate into a picture of the postgres logo, as can be seen here: https://buildkite.com/matrix-dot-org/synapse/builds/16518#4b64142c-9740-4104-9009-7df4230abd54

In case you were wondering :blush:

Reply:
Alas, GHA does not support custom emoji names for jobs ☹️